src/components/color-picker/color-picker.tsx

```diff
@@ -0,0 +1,1295 @@
+    const newColorVariants = fromStringToColorVariants(
+      `rgb(${Math.round(rgb[0])}, ${Math.round(rgb[1])}, ${Math.round(rgb[2])})`
+    );
+
+    if (newColorVariants) {
+      this.colorVariants = newColorVariants;
+      this.#emitColorChange();
+    }
+  };
+
+  // Handles alpha input changes
+  #handleAlphaInputChange = (event: Event): void => {
+    const input = event.target as HTMLInputElement;
+    const value = parseFloat(input.value) || 0;
+    const alphaValue = Math.max(0, Math.min(100, value)) / 100;
+
+    // Get current RGB values
+    const rgbMatch = this.colorVariants.rgb.match(RGB_REGEX);
+    if (!rgbMatch) {
+      return;
+    }
+
+    const [, r, g, b] = rgbMatch;
+    const newColorVariants = fromStringToColorVariants(
+      `rgba(${r}, ${g}, ${b}, ${alphaValue})`
+    );
+
+    if (newColorVariants) {
+      this.colorVariants = newColorVariants;
+      this.#emitColorChange();
+    }
+  };
+
+  // Renders the color field component
+  #renderColorField = (): Element => {
+    // Use the current color value from colorVariants
+    const currentColorValue = this.#getColorInOriginalFormat(
+      this.colorVariants
+    );
+    const { accessibleName, text } = this.translations;
+
+    return (
+      <div class="color-field-container">
+        {text.colorFieldLabel && (
+          <label
+            htmlFor="color-field"
+            part={COLOR_PICKER_PARTS_DICTIONARY.COLOR_FIELD_LABEL}
+          >
+            {text.colorFieldLabel}
+          </label>
+        )}
+        <ch-color-field
+          id="color-field"
+          class="color-picker-field"
+          aria-disabled={this.disabled ? "true" : null}
+          aria-readonly={this.readonly ? "true" : null}
+          disabled={this.disabled}
+          readonly={this.readonly}
+          part={COLOR_PICKER_PARTS_DICTIONARY.COLOR_FIELD}
+          value={currentColorValue}
+          step={this.colorFieldStep}
+          accessibleName={accessibleName.colorFieldControl}
+          accessibleRoleDescription={accessibleName.colorFieldDescription}
+          onInput={this.#handleColorFieldChange}
+        ></ch-color-field>
+      </div>
+    );
+  };
+
+  // Renders the hue slider component
+  #renderHueSlider = (): Element => {
+    // Parse HSV string to get individual values
+    const hsvValues = fromHsvStringToHsvColor(this.colorVariants.hsv);
+    const { h: hueValue } = hsvValues;
+    const { accessibleName, text } = this.translations;
+
+    return (
+      <div class="slider-group">
+        {text.hueChannelLabel && (
+          <label
+            htmlFor="hue-slider"
+            part={COLOR_PICKER_PARTS_DICTIONARY.HUE_SLIDER_LABEL}
+          >
+            {text.hueChannelLabel}
+          </label>
+        )}
+        <ch-slider
+          id="hue-slider"
+          accessibleName={
+            !text.hueChannelLabel ? accessibleName.hueSliderControl : undefined
+          }
+          aria-disabled={this.disabled ? "true" : null}
```

Comment:
Just as we don't add aria-disabled="true" when we using a disabled input field, we don't have to add it to the `ch-slider` either. This feature must always be integrated into the component. 

```suggestion
```

Reply:
Same for all.